Patterns/English/English-TimeZone.yaml

```diff
@@ -4,8 +4,301 @@ DirectUtcRegex: !simpleRegex
   def: \b(utc|gmt)(\s*[+\-\u00B1]?\s*[\d]{1,2}(\s*:\s*[\d]{1,2})?)?\b
 AbbreviationsRegex: !simpleRegex
   def: (\b)(AFT|AKST|AKDT|ABST|AST|ARBST|ARST|ART|ADT|ACST|AEST|AEDT|AZT|AZST|AZOT|AZOST|BST|CCST|CT|CVT|AMT|AMST|ACDT|CAST|CADT|BTT|CBT|CBST|CET|CEST|SBT|CST|CDT|CSTM|CDTM|BIT|ET|EAT|EET|EEST|ESAT|ESAST|EST|EDT|ESTM|EDTM|EGST|YEKT|FJT|FJST|GET|GMT|GNST|GNDT|GST|GTBST|HAST|HST|HADT|HDT|IST|IRST|IRDT|ISST|ISDT|JST|JDT|PETT|KST|LINT|MAGT|MAGST|MUT|MAT|MEST|MVST|MOST|WEST|MST|MDT|MSTM|MDTM|MYST|NCAST|NMST|NMDT|NPT|NZT|NZST|NZDT|NST|NDT|IRKT|KRAT|PSAT|PSAST|PST|PDT|PSTM|PDTM|PKT|PYT|PYST|RST|MSK|SAEST|SAPST|SAWST|SST|SMST|THA|SNST|SAST|SLT|SGT|TIST|TAST|TADT|TST|TOT|TOST|TRT|ULAT|UTC|VET|VLAT|AWST|WAT|ECT|WAST|WET|WPST|YAKT|PDST|PT|AWDT|COT|HKT)(\b)
+AbbreviationsList: !list
```

Comment:
Is FullNameRegex and AbbreviationsRegex still used in some piece of the code?
If not, I suggest to remove these two Regexes from this yaml file.
Thanks.

Reply:
Build failed

Reply:
C# does not currently use these two regexes, but Java and C# are not synchronized, so they are not deleted for now. These two regexes should be deleted after Java implements StringMatcher.

Reply:
@Sothan I left a comment in the review. Can you also implement the Java matcher?

Reply:
Sorry, the StringMatcher is already implemented in Java, I will refactor this PR ASAP.